packages/gatsby/src/utils/node-manifest.ts

```diff
@@ -311,6 +316,25 @@ export async function processNodeManifest(
   return finalManifest
 }
 
+function nodeManifestSortComparerAscendingCreatedAt(a, b): number {
+  /**
+   * Prioritize node manifests that do not have an updatedAtUTC
+   */
```

Comment:
Done! :white_check_mark: 

And actuallly decided to flip the logic and prioritize manifests that have `updatedAtUTC` over those that do not because we actually know when they were created as opposed to the others that could have been created at any time (i.e. the might be new but they also might be very old)